static/js/subs.js

```diff
@@ -737,6 +740,14 @@ export function launch(section) {
     if (!get_active_data().id) {
         $("#search_stream_name").trigger("focus");
     }
+    tippy_tooltip_subunsub();
+}
+
+function tippy_tooltip_subunsub() {
+    // Set tooltips for subs-tick
+    tippy(".place-top-subunsub", {
+        placement: "top-end",
+    });
```

Comment:
This seems like it'd be better done in generic code. than calling this function every time we render a row.

Reply:
What i understood from this is that I should do this thing is some generic place where it simply applies to all cases.
Well, I tried to find a generic code to place the call there but found these 2 separate cases(launch and update), I'll try go find and place it in a generic code again. :+1:

Reply:
@amanagr Any suggestion as to where to place the call?